src/index.ts

```diff
@@ -5,7 +5,8 @@ function load() {
   try {
     return require('../build/Release/mongocrypt.node');
   } catch {
-    return require('../build/Debug/mongocrypt.node');
+    // eslint-disable-next-line no-console
+    console.log('Could not load the native module mongocrypt.node');
```

Comment:
The Debug native binding is never published by us, so no reason to attempt to include it in the published package and generate a warning from Webpack.

Reply:
```suggestion
     try {
      return require('../build/Debug/mongocrypt.node');
    } catch (error) {
      throw error;
    }
```

It is kinda nice to have the code set up to pull in the debug build when dev-ing locally. So all we need here is a `try {}` around the `require`

You may see "Module not found: Error: Can't resolve" but it is printed as a "WARNING" and does not stop the build and that's the intended behavior

Reply:
I had a conversation with Bailey about this and we thought it was more developer friendly to not have a warning even in the event of a valid bundle. I'm on that side that the bundle should be warning-free, since the debug aspect is only for ourselves and would never be published.

Reply:
Also just to clarify with the double try/catch, Webpack will still issue a warning about the Debug module not being available, though it will not fail. I think it's a better developer experience that if they bundle the module correctly, no warning is issued.

Reply:
The warning is desirable IMO because each warning alerts you to what your bundle is missing, you can squash them in your webpack config if you don't want to see them and it creates an intentional "paper trail" of the resolutions you don't care about.

We may not ship Debug but it can still be built and imported if Release is removed. 

If we really feel strongly about not having Debug that's fine but we shouldn't introduce a console.log on import when this is missing, we can just throw inside this catch. The module is going to throw anyway when `mc` is accessed during `extends mc.MongoCrypt`

Reply:
Yeah, definitely feelings strongly that:

1. We should support the debug-mode addon, running `npm install --debug --build-from-source ...` isn't that hard and it's usually what a (sufficiently experienced) user will do to start diagnosing issues with native addons
2. `require()`ing a native addon package should throw if the addon itself is unavailable – that's just the general expectation for how addon packages behave
3. A warning at runtime printed via `console.error()` is much more disruptive than a warning in webpack. Most webpack configs will result in a bunch of warnings being printed, that's unfortunately more or less a fact of life

Reply:
I've updated to throw now if both modules are missing.